models/gan.py

```diff
@@ -380,13 +393,14 @@ def forward(self, input, style):
 
 
 class ToXYZ(nn.Module):
-    def __init__(self, in_channel, style_dim, upsample=True, blur_kernel=[1, 3, 3, 1],
+    def __init__(self, in_channel, style_dim, upsample=True, blur_kernel=(1, 3, 3, 1),
```

Comment:
Why change from list to tuple

Reply:
No real reason, it is better convention to pass non mutable arguments as default arguments.